src/pages/LogInWithShortLivedTokenPage.js

```diff
@@ -38,37 +39,58 @@ const propTypes = {
         /** The authToken for the current session */
         email: PropTypes.string,
     }),
+
+    /** Beta features list */
+    betas: PropTypes.arrayOf(PropTypes.string),
 };
 
 const defaultProps = {
     route: {
         params: {},
     },
     session: {},
+    betas: null,
 };
+
 class LogInWithShortLivedTokenPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {run: false};
```

Comment:
I don't really get `run` for the state variable, should it be `hasRan`?

Reply:
Yes, hasRun would be a better name,

Reply:
NAB